server/svix-server/tests/e2e_attempt.rs

```diff
@@ -145,3 +150,130 @@ async fn test_list_attempts_by_endpoint() {
     receiver_1.jh.abort();
     receiver_2.jh.abort();
 }
+
+async fn test_attempt_retries(endpoint: &str, expected_status: i16) {
+    let _ = dotenv::dotenv();
+    let mut cfg = svix_server::cfg::load().unwrap().as_ref().clone();
+    cfg.queue_type = svix_server::cfg::QueueType::Memory;
+    cfg.retry_schedule = (0..2)
+        .into_iter()
+        .map(|_| Duration::from_millis(1))
+        .collect();
+
+    let (client, _jh) = start_svix_server_with_cfg(&cfg);
+
+    let app_id = create_test_app(&client, "app1").await.unwrap().id;
+
+    let endp_id = create_test_endpoint(&client, &app_id, endpoint)
+        .await
+        .unwrap()
+        .id;
+
+    let msg = create_test_message(&client, &app_id, serde_json::json!({"test": "data1"}))
+        .await
+        .unwrap();
+
+    for i in &cfg.retry_schedule {
+        sleep(*i).await;
+    }
+
+    run_with_retries(|| async {
+        let list: ListResponse<MessageAttemptOut> = client
+            .get(
+                &format!("api/v1/app/{}/attempt/msg/{}", app_id, &msg.id),
+                StatusCode::OK,
+            )
+            .await
+            .unwrap();
+
+        if list.data.len() != cfg.retry_schedule.len() + 1 {
+            anyhow::bail!("Attempt count does not match retry_schedule length");
+        }
```

Comment:
```suggestion
        assert_eq!(list.data.len(), cfg.retry_schedule.len() + 1);
```
Use asserts in tests.